ui/app/templates/components/cluster-info.hbs

```diff
@@ -1,30 +1,30 @@
 <div class="popup-menu-content">
   <div class="box">
-    {{#unless version.isOSS}}
-      {{#if (and activeCluster.unsealed auth.currentToken)}}
-        {{#unless cluster.dr.isSecondary}}
+    {{#unless this.version.isOSS}}
+      {{#if (and @cluster.unsealed this.auth.currentToken)}}
```

Comment:
Based on looking at the code, I'm not sure `activeCluster` as defined on (old) status-menu.js is exactly swappable with `currentCluster.cluster` which is what's being passed into the component here. My guess is that with the full model (activeCluster) we get the replication info and we may not on the cluster service version, but we'll need to investigate

Reply:
I added back the active cluster for now.
In meantime, I will dig further and if required get in touch with Matthew. We can clean up later if required.